.ebextensions/01_packages.config

```diff
@@ -0,0 +1,28 @@
+packages:
+   yum:
+     git: []
+     python36-virtualenv: []
+     gcc: []
+     gcc-c++: []
+     postgresql96-devel: []     
+     httpd24-devel: []
+files:
+  "/tmp/update-wsgi.sh" :
+    mode: "000755"
+    owner: root
+    group: root
+    content: |
+      # update mod_wsgi
+      cd /tmp
+      wget -q "https://github.com/GrahamDumpleton/mod_wsgi/archive/4.4.21.tar.gz" && \
```

Comment:
Is wget from github really the preferred way of doing this? No package management system, no hash checking or anything to ensure that we're getting what we expect to get?

This is kind of YOLO for my tastes, especially with root ownership, but if it's the only option then 🤷‍♀️ I guess.

Reply:
I'm hoping this changes with a newer release and we don't need it. But a lot of people seem to be using it.. https://serverfault.com/questions/884469/mod-wsgi-call-to-site-addsitedir-failed-on-aws-elastic-beanstalk-python-3

Reply:
OK. I'll add a comment so future-us isn't confused about that.